migrations/20200429204207-move-security-code-to-recovery-methods.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+module.exports = {
+    up: (queryInterface, Sequelize) => {
+        return queryInterface.sequelize.transaction(transaction => {
+            return Promise.all([
+                queryInterface.addColumn('RecoveryMethods', 'securityCode', {
+                    type: Sequelize.STRING
+                }, {transaction}),
+                queryInterface.removeColumn('Accounts', 'securityCode', {transaction}),
+                queryInterface.removeColumn('Accounts', 'confirmed', {transaction}),
+            ]);
+        });
+    },
+
+    down: (queryInterface, Sequelize) => {
+        return queryInterface.sequelize.transaction(transaction => {
```

Comment:
@Patrick1904 tip (no action required): we don't really use `down` migrations anywhere in our process.